pkg/agent/openflow/pipeline_windows.go

```diff
@@ -281,5 +281,24 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 	}
+	if c.encapMode.SupportsNoEncap() {
+		// If NoEncap is enabled, the return traffic has symmetric path as request one. After entering uplink, the return
```

Comment:
The return traffic goes the symmetric path as the requests.

After inputting from uplink,

Reply:
Updated.